src/proposal/IProposal.sol

```diff
@@ -107,25 +104,9 @@ interface IProposal is IModuleManager {
         address rebasingFundingManagerAddress
     ) external view returns (bool);
 
-    /// @notice Verify whether the given address is the payment client
-    function verifyAddressIsPaymentClient(address paymentClientAddress)
+    /// @notice Verify whether the given address is an IAuthorizer module
```

Comment:
What about the other major modules?
You changed the Authorizer Module, but not Payment Processor or fundingmanager